packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/relativeScopes/clearOneLines.yml

```diff
@@ -1,6 +1,6 @@
 languageId: plaintext
 command:
-  spokenForm: change one lines
+  spokenForm: change line forward
```

Comment:
There's no way to disambiguate "one lines" vs "line forward". I think for the singular using line forward is more intuitive.

Reply:
Agreed.  Maybe worth having spoken form test for this one as well.  Is it supposed to be plural?